client/src/components/ImagePanel.jsx

```diff
@@ -4,19 +4,31 @@ class ImagePanel extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      //  initial state
```

Comment:
could clean this up a bit with destructuring. Not a priority though, more of a clean-up note